cmd/oci-image-tool/validate.go

```diff
@@ -67,6 +68,11 @@ func newValidateCmd(stdout, stderr *log.Logger) *cobra.Command {
 		`A set of refs pointing to the manifests to be validated. Each reference must be present in the "refs" subdirectory of the image. Only applicable if type is image or imageLayout.`,
 	)
 
+	cmd.Flags().BoolVarP(
+		&v.verbose, "verbose", "v", false,
+		fmt.Sprintf(`Output verbosely, including all objects digest to be validated. If unset, oci-image-tool will only output a OK or failed result.`),
+	)
```

Comment:
Ocitools uses logrus and `--log-level` (e.g. see opencontainers/ocitools#97).  I'd recommend we go that route instead of carrying a `verbose` boolean around with us.